db/migrate/20230711150925_create_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.text :Name
+      t.text :Bio
+      t.integer :PostsCounter
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ OPTIONAL] I just noticed that in your code for creating the "users" table, there are a few areas that could be improved. Firstly, it's recommended to follow naming conventions, so instead of using `Name`, `Bio`, and `PostsCounter`, consider using `name`, `bio`, and `posts_counter` respectively. But the code for generating timestamps looks fine. :wink:

Reply:
Fixed, thanks